metrics/web/docs/features/firestore_emulator_for_driver_tests/01_firestore_emulator_for_driver_tests.md

```diff
@@ -26,13 +36,12 @@ _**Note:**
 We are not configuring the [Firebase Auth emulator](https://firebase.google.com/docs/emulator-suite/connect_auth) because it is not available in the version of the [firebase_auth](https://pub.dev/packages/firebase_auth) package we are using. So, during tests, we authenticate via the production Firebase Auth service._
 
 ### Requirements
-
 > Define requirements and make sure that they are complete.
 
```

Comment:
:tw: It would be great to introduce a requirements list here. What do you think?